modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -47,6 +48,112 @@ next@docs::server_release_notes.adoc, next@docs_main::server_release_notes.adoc
+
+The admin can setup external storages to be used by specific users and groups. When a LDAP group was setup, there were some issues so users belonging to that group weren't able to access the external storage even though they should. Now, users belonging to LDAP groups can access external storages configured to be accessed by those groups. https://github.com/owncloud/core/pull/41063[#41063]
+
+=== Disallow Browsers to Translate the Frontend
+
+Web frontend offers a lot of proper translations. Browser capabilities are not needed. https://github.com/owncloud/core/pull/41067[#41067]
+
+=== Limit Performance Impact when Version Metadata is Enabled
+
+Negative performance impact when `file_storage.save_version_metadata` is enabled has been resolved. https://github.com/owncloud/core/pull/41069[#41069]
+
+=== Proper Error Handling when Deleting Users or Groups
+
+In case of an error when deleting a user or a group no proper feedback was given to the user. This is now fixed. https://github.com/owncloud/core/pull/41077[#41077]
+
+=== Account Terms Can be Removed if They Differ From the Ones Stored
+
+Search terms for the account can now be removed from the DB if they differ from the ones stored in the DB. This can happen with the user_ldap app, when user search attributes are removed from the connection configuration in the LDAP wizard. https://github.com/owncloud/core/pull/41120[#41120]
+
+=== Add Compatibility for richdocuments.document.index Route
+
+Because of a change first introduced in richdocuments 4.1.0, it was necessary to add compatibility for the richdocuments.document.index route. https://github.com/owncloud/core/pull/41161[#41161]
+
+=== Decommission of Template Editor App
+
+App has been deprecated and themes should be rather used to change mail templates. https://github.com/owncloud/core/pull/41168[#41168]
+
+=== The `user:list -a` occ Command now Correctly Displays the User's Creation Time
+
+Previously, the `user:list -a` occ command was not correctly returning the user's creation time but rather the path to the user's home directory. This has been now fixed. https://github.com/owncloud/core/pull/41125[#41125]
+
+=== Set oc_file_locks.id to bigint
+
+In rare cases, the id column in the file_locks table could hit its limit (defined as int(10) unsigned) as it is set as autoincrement. This is now set to bigint. https://github.com/owncloud/core/pull/41158[#41158]
+
+=== Store User Information in Explicit Variable
+
+Before user information was stored in the browser global object. In some rare cases browsers seemed to loose data stored in the global object. This is fixed now. https://github.com/owncloud/core/pull/41054[#41054]
+
+=== No Activities on Rejected Shares
+
+As soon as a user has rejected a share no activities within this share are now reported via the activity app. https://github.com/owncloud/core/pull/41078[#41078]
+
+=== CalDAV Query Where the Time Range Is not Given
```

Comment:
Would leave this out, complete list is in changelog. Focus on the once relevant for admin and content collaboration use case.

Reply:
Why? Caldav and cardav is a major reason for the community to use own cloud.
Release notes are a list of facts. No marketing tool.